docs/public/static/diffs/dev-launcher/podfile.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/ios/Podfile b/ios/Podfile
+index 8c935a2..1b1f955 100644
+--- a/ios/Podfile
++++ b/ios/Podfile
+@@ -6,10 +6,14 @@ platform :ios, '10.0'
+ 
+ target 'expodevexample' do
+   use_unimodules!
++  
++  pod 'EXDevMenu', path: '../node_modules/expo-dev-menu', :configurations => :debug
++  pod 'expo-dev-launcher', path: '../node_modules/expo-dev-launcher', :configurations => :debug
```

Comment:
Do we need this? They're already autolinked, right?
Also, I think it may make sense to unify pod names of both 😉

Reply:
They're auto-linked, but I believe that the `react-native-cli` wasn't published. So, they won't be removed in release builds.